DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/tgen_utils.py

```diff
@@ -743,3 +743,23 @@ async def tgen_utils_send_arp(device, config):
     if out[0][device.host_name]["rc"] != 0:
         device.applog.warning(f"Some arps did not reach their destination\n{out}")
     return out[0][device.host_name]["result"]
+
+
+async def tgen_utils_update_l1_config(device, ixia_ports, speed=None, autoneg=True, duplex="Full"):
```

Comment:
Please rename `ixia_ports` to `ports`. IMO it is obvious that these are TGen ports. And adding "ixia_" prefix is not acceptable since this is an abstraction, and we can have non-Ixia backend.

Reply:
Named it first time `ports` but in `traffic.yaml` and `ixia.yaml` wee already have param with the name `ports` and it would not allow to generate the cod with non-unique param.  Therefore named it `ixia_ports`. Suggest naming it `tgen_ports`